cmd/tapcli/assets.go

```diff
@@ -858,6 +860,75 @@ func burnAssets(ctx *cli.Context) error {
 	return nil
 }
 
+var listBurnsCommand = cli.Command{
+	Name:  "listburns",
+	Usage: "list burnt assets",
+	Description: `
+	List assets that have been burned by this daemon. These are assets that
+	have been destroyed and are no longer spendable.
+
+	Some filters may be used to return more specific results.
+	`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  assetIDName,
+			Usage: "the asset ID of the burnt asset",
+		},
+		cli.StringFlag{
+			Name:  assetGroupKeyName,
+			Usage: "the group key of the burnt asset",
+		},
+		cli.StringFlag{
+			Name: anchorTxidName,
+			Usage: "the txid of the transaction the burn was " +
+				"anchored to",
+		},
+	},
+	Action: listBurns,
+}
+
+func listBurns(ctx *cli.Context) error {
+	if ctx.NArg() != 0 || ctx.NumFlags() == 0 {
```

Comment:
I think calling this without any flags should be okay, if I just want all burns?